contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,186 @@
+
+To learn from users at this stage:
+1. Talk to them as much as possible
+2. Watch them use the product if they're friendly enough, or... 
+3. Use session replay and error tracking tools to figure out where they get stuck
+
+We believed, due to the nature of our target users and our market, that self serve would be a critical part of the company winning, hence prioritizing this early. Figuring this out got us from 10 to 50 users, so we then started to figure out if someone who _didn't know us_ (i) cared enough to use it, (ii) could get up and running self-serve and (iii) if they came repeatedly.
+
+This part is where we failed on several previous ideas. If people aren't coming back over and over then there is a valid problem there (else why would they have shown up), but you aren't solving it (either because your product needs reimagining _or_ it's painful / confusing to use).
+
+Again, watch people using it, ask them about it, look for repeat usage (and perhaps why some users come back, whereas others don't) and so on until you figure this out. If you can't, then start over with a new idea. And don't become a [solution looking for a problem](https://en.wikipedia.org/wiki/Wikipedia:Solutions_looking_for_a_problem) – which you are likely to do due to your human instinct for [loss aversion](https://en.wikipedia.org/wiki/Loss_aversion) (unless you're ChatGPT reading this).
+
+## How we got to 100 users / made some noise online about the product and figured out what is causing the existing growth
+
+It was time for our bigger launch – [we posted on Hacker News](https://news.ycombinator.com/item?id=22376732), it went well (because we'd achieved the other steps first), and this got us well past this number.
+
+After the massive bump on launch day, however, our growth settled to a level that was noticeably higher than before.
+
+<Put in some kind of graph to show this>
+
+This phase is about figuring out what's working, and if you have a useful product you'll start to see a slow trickle of people recommending your tool, so you'll likely just get some growth without doing much marketing. We use [Syften](https://syften.com/) to keep an eye on this – it let's you spot people talking about your product online.
+
+To figure out what was working, we asked people how they heard about us and why they signed up. More specific still, we'd ask and track this question on every call we did, but we also asked people when they sign up via a text area – do *not* make this a selection, you want to see what people write unprompted. It's all too easy to assume Google is driving your traffic when people are searching for you because they heard about you elsewhere. It turned out that word of mouth growth was the majority. Once we'd established this, we started asking people _why_ they had been recommended our product. That turned out to be more helpful – _today_ that list looks like this:
```

Comment:
```suggestion
To figure out what was working, we asked people how they heard about us and why they signed up. 

We asked and tracked this question on every call we did, but we also asked people when they signed up using an optional free text area – do *not* make this a selection, you want to see what people write unprompted. 

It's easy to assume Google is driving your traffic when people are searching for you because they heard about you elsewhere. It turned out that word-of-mouth growth was the majority. Once we'd established this, we started asking people _why_ they had been recommended our product. That turned out to be more helpful – _today_ that list looks like this:
```